src/Form.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import Input from "./Input";
+import { createNewConference } from "./api";
+import { useHistory } from "react-router-dom";
+
+const Form = () => {
+  let history = useHistory();
+  const [conference, setConference] = useState({
+    name: "",
+    dateTime: "",
+    city: "",
+    description: "",
+    topic: ""
+  });
+
+  const handleChange = event => {
+    const value = event.target.value;
+    setState({ ...conference, [event.target.name]: value });
+  };
+
+  const submitForm = async () => {
+    const newConference = {
+      ...conference,
+      dateTime: conference.dateTime + ":00Z"
+    };
+    try {
+      await createNewConference(newConference);
+      history.push("/");
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <form
+      onSubmit={ev => {
+        ev.preventDefault();
+        submitForm();
+      }}
+    >
+      <Input
+        id="name"
```

Comment:
May want to remove this id and the one on the submit we tried for testing purposes!